blueocean-dashboard/src/main/js/components/PullRequest.jsx

```diff
@@ -2,7 +2,7 @@ import React, { Component, PropTypes } from 'react';
 import { LiveStatusIndicator, ReadableDate } from '@jenkins-cd/design-language';
 import Extensions from '@jenkins-cd/js-extensions';
 import RunPipeline from './RunPipeline.jsx';
-import { buildRunDetailsUrl } from '../util/UrlUtils';
+import { getLocation } from '../util/UrlUtils';
```

Comment:
is this the one getLocation to rule them all? or does it just replace run details url?